lms/envs/common.py

```diff
@@ -45,7 +45,7 @@
     ENTERPRISE_OPERATOR_ROLE
 )
 
-from openedx.core.constants import COURSE_KEY_REGEX, COURSE_KEY_PATTERN, COURSE_ID_PATTERN
+from openedx.core.constants import COURSE_KEY_REGEX, COURSE_KEY_PATTERN, COURSE_ID_PATTERN, COURSE_ID
```

Comment:
why are we importing `COURSE_ID` here?

Reply:
same idea, as other regexes are being imported here